### ISSUE TITLE
Please explain --pretransform-ckpt-path

### ISSUE THREAD

**Issue by neops1** [Stability-AI/stable-audio-tools#105]
Which ckpt should I use in  --pretransform-ckpt-path to train my model from scratch? I have tried weights from DAC but without success. The latest version uses oobleck as an encoder, does it need some kind of pretransform?

**Comment by BingliangLi** [comment]
Yes, this is equivalent to the VAE part in Stable Diffusion, you need to train it yourself. However there is a open source version just released, maybe you could try the VAE from this: https://huggingface.co/stabilityai/stable-audio-open-1.0
